packages/rich-text/src/plugins/PasteHTML/utils/sanitizeHTML.ts

```diff
@@ -20,6 +20,7 @@ export const sanitizeHTML = (html: string): string => {
     new DOMParser().parseFromString(html, 'text/html')
   );
 
-  // Remove whitespace between tags
-  return doc.body.innerHTML.replace(/>\s+</g, '><');
+  return doc.body.innerHTML
+    .replace(/>\s+</g, '><') // Remove whitespace between tags
+    .replace(/(.*)<div.*>(<table.*<\/table>)<\/div>(.*)/g, '$1$2$3'); // remove div containers from tables
```

Comment:
```suggestion
    .replace(/<div.*>(<table.*<\/table>)<\/div>/g, '$1'); // remove div containers from tables
```

Reply:
I need what comes before and after the table container because it can include other elements such as lists, headings, paragraphs, etc.

Reply:
I thinks js leaves those sections there, only replacing the match